site/client/blocks/ProminentLink/ProminentLink.tsx

```diff
@@ -0,0 +1,112 @@
+import * as React from "react"
+import ReactDOM from "react-dom"
+import { Grapher } from "site/client/Grapher"
+import { observer } from "mobx-react"
+import { computed } from "mobx"
+import {
+    strToQueryParams,
+    queryParamsToStr,
+    splitURLintoPathAndQueryString,
+    QueryParams
+} from "utils/client/url"
+import { EntityUrlBuilder } from "charts/ChartUrl"
+import { union, isEmpty } from "charts/Util"
+
+const PROMINENT_LINK_CLASSNAME = "wp-block-owid-prominent-link"
```

Comment:
nit: can we export this or import it from somewhere since it's used in many places?

Reply:
I'm not sure I understand. This constant is only used once on line 95.

Reply:
Sorry, I meant it's used in 3-4 places in other files besides this one.